rudder-server-relay/SOURCES/rudder-pkg

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/python
+
+"""
+Rudder package manager
+
+Usage:
+    rudder-pkg install-file <package.rpkg>
+    rudder-pkg list
+    rudder-pkg remove <package>
+    rudder-pkg check-compatibility
```

Comment:
check-compatibility looks more like "disable-incompatible". It can also restart the webapp which is unexpected from a "check" command